test/e2e/data/infrastructure-vsphere-supervisor/main/ownerrefs-finalizers/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+patchesStrategicMerge:
+  - ../commons/cluster-resource-set-label.yaml
+  - ../commons/cluster-network-CIDR.yaml
+  - ../commons/cluster-resource-set-csi-insecure.yaml
```

Comment:
base already adds this patches if I did see it right

we could maybe even just generate the tempate out of base without needing this kustomization?

```suggestion
```

Reply:
Yeah we don't need these. Which also means we don't need a different template. We can just use the base / default template for the test instead

Reply:
we need separated template because the corresponding govimovi template is different from govimomi base.